vpr/SRC/route/route_timing.c

```diff
@@ -514,27 +561,110 @@ bool timing_driven_route_net(int inet, int itry, float pres_fac, float max_criti
 	 * case the rr_graph is disconnected and you can give up. If slacks = NULL, *
 	 * give each net a dummy criticality of 0.									*/
 
-	unsigned int ipin;
-	int num_sinks, itarget, target_pin, target_node, inode;
-	float target_criticality, old_total_cost, new_total_cost, largest_criticality,
-		old_back_cost, new_back_cost;
-	t_rt_node *rt_root;
-	
-	struct s_trace *new_route_start_tptr;
-	int highfanout_rlim;
-
 	/* Rip-up any old routing. */
+	// convert the previous iteration's traceback into the starting route tree for this iteration

```

Comment:
It would be useful at this point to give a higher level comment describing what we are trying to accomplish here.
i.e. We will completely rip up nets below a certain threshold, but re-use the legal parts of the previous iterations routing